kolibri/core/content/models.py

```diff
@@ -446,6 +446,41 @@ def _hex_uuid_str():
     return str(uuid.uuid4().hex)
 
 
+class ContentRequestManager(models.Manager):
+    request_type = None
+
+    def get_queryset(self):
+        """
+        Automatically filters on the request type for use with proxy models
+        :rtype: django.db.models.QuerySet
+        """
+        queryset = super(ContentRequestManager, self).get_queryset()
+        if self.request_type is not None:
+            queryset = queryset.filter(type=self.request_type)
+        return queryset
+
+    def propagate_removal(self, contentnode_ids):
```

Comment:
While considering code organization and interacting with the models, having something like this on the base manager feels straightforward. Although, I do think it is slightly awkward because it is inherited by the individual typed managers and it affects both downloads and removals

Reply:
You could separate this into a mixin and define a new manager class purely for the base model, such that this is only available there.

Reply:
Discussed with @bjester and am just going to move this into a separate utility function to avoid confusion here.

Reply:
Updated!